spec/requests/v1/developers_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+require 'spec_helper'
+require 'support/shared_context_for_api.rb'
+
+describe 'Api::V1::DevelopersController' do
+  include_context 'api'
+
+  describe 'GET /api/v1/developers/:developer_id' do
+    before do
+      @developer = FactoryBot.create(:developer,
+                                     languages: [FactoryBot.create(:language)],
+                                     programming_languages: [FactoryBot.create(:programming_language)])
+    end
+
+    context 'when the developer exists' do
+      # let(:expected_response) do
+      #   {
+      #     id: @developer.id,
+      #     email:  @developer.email,
+      #     languages: @developer.languages.to_json(only: :code),
+      #     progamming_languages: @developer.programming_languages.to_json(only: :name)
+      #   }
+      # end
+      let(:expected_response) do
```

Comment:
I wonder if the response complies with json_api, which is written in the requirement.
It doesn't look so.

Reply:
Thanks you very much for helping me review this PR!
I have made change according to review